### ISSUE TITLE
[Bug] it's broken if specific strings are included.

### ISSUE THREAD

**Issue by yahsan2** [delashum/obsidian-checklist-plugin#170]
I got this error when there is string including `- [ ] [ ][[link]] title` in check list.
> Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'filePath')

<img width="1121" alt="Screenshot 2023-02-16 at 23 36 08" src="https://user-images.githubusercontent.com/610752/219394392-0ed9317b-69ac-43f2-8946-48dd3c355315.png">

---
macOS:    13.2.1 
Obsidian: Version 1.1.9

**Comment by Bill77** [comment]
I hit the same error, but it was because I aliased my link. It broke on `|`. 

``` md
// Works
- [ ] Something [[link]]

// Doesn't work
- [ ] Something [[link | A]]
```

> Oddly your example seems to works for me (maybe there's another plugin you have?)
> `- [ ] Something [[link]] test`
> `- [ ] [ ] Something [[link]] test`
>`- [ ] [ ] [[link]] test`

**Comment by ryardley** [comment]
Aliases actually work - the work around I have found is to not include spaces: 

```
// Works
- [ ] [[20230507 Some other document|Some doc]]

// Doesn't Work
- [ ] [[20230507 Some other document| Some doc]]

// Doesn't Work
- [ ] [[20230507 Some other document |Some doc]]


```

**Comment by Bill77** [comment]
Thanks @ryardley, yes that format without spaces works - I can live with that as a solution. But it'd be nice if it was a bit more forgiving on the aliases. 😄

**Comment by delashum** [comment]
Fixed in 2.2.13
